contrib/window_tshirt_sizes/src/sizes.cpp

```diff
@@ -0,0 +1,139 @@
+
+#include <cassert>
+#include <stdio.h>
+
+class SDL_Point {
+public:
+	int x            = 0;
+	int y            = 0;
+	const char* name = nullptr;
+	constexpr SDL_Point(const int _x, const int _y, const char* _name = nullptr)
+	        : x(_x),
+	          y(_y),
+	          name(_name){};
+};
+
+constexpr SDL_Point eight_by_five_ratio = {8, 5, "widescreen"};
+constexpr SDL_Point four_by_three_ratio = {4, 3, "standard"};
```

Comment:
I think we should call these what they really are:

- standard -> "1.2 pixel aspect ratio"
- widescreen -> "1.0 pixel aspect ratio"

I *assume* that's what you meant here; for the aspect-ratio corrected case we use 4:3 with with 1.2 PAR, then for the aspect-ratio uncorrected case we use 1.0 PAR (e.g. 320/200 = 8/5 with square pixels).

It gets confusing and tricky to convey the right meaning, and I'm not saying the above names I proposed are the best, but I find "standard" and "widescreen" confusing.